pyface/ui/qt4/init.py

```diff
@@ -14,22 +14,10 @@
 
 from traits.trait_notifiers import set_ui_handler, ui_handler
 
-from pyface.qt import QtCore, QtGui, qt_api
+from pyface.qt import QtCore, QtGui
 from pyface.base_toolkit import Toolkit
 from .gui import GUI
 
-if qt_api == "pyqt":
-    # Check the version numbers are late enough.
-    if QtCore.QT_VERSION < 0x040200:
-        raise RuntimeError(
-            "Need Qt v4.2 or higher, but got v%s" % QtCore.QT_VERSION_STR
-        )
-
-    if QtCore.PYQT_VERSION < 0x040100:
-        raise RuntimeError(
-            "Need PyQt v4.1 or higher, but got v%s" % QtCore.PYQT_VERSION_STR
-        )
```

Comment:
seems as though we weren't supporting these anyway, just raising saying version is too old